neuromation/cli/job.py

```diff
@@ -75,6 +77,11 @@
 NEUROMATION_HOME_ENV_VAR = "NEUROMATION_HOME"
 RESERVED_ENV_VARS = {NEUROMATION_ROOT_ENV_VAR, NEUROMATION_HOME_ENV_VAR}
 
+DEFAULT_JOB_LIFE_SPAN = "1d"
+REGEX_JOB_LIFE_SPAN = re.compile(
+    r"^((?P<d>\d+)d)?\s*((?P<h>\d+)h)?\s*((?P<m>\d+)m)?\s*((?P<s>\d+)s)?$"
```

Comment:
Please drop the optional spaces.
I know you can support this mode easily but the case doesn't make sense: wrapping an option value in quotes is cumbersome.
I don't see the motivation for support of spaces except for *just we can do it*.

Reply:
ok